linkerd/examples/k8s.yaml

```diff
@@ -0,0 +1,19 @@
+admin:
+  port: 9990
+
+namers:
+- kind: io.l5d.experimental.k8s
+  host: localhost
+  port: 443
+  tls: true
+  tlsWithoutValidation: true
+  authTokenFile: linkerd/examples/io.l5d.k8s/kube_token
```

Comment:
That's a good point.  I don't think any of the examples other than acceptance-test.yaml are tested.  I created https://github.com/BuoyantIO/linkerd/issues/280 to track.